### PR TITLE
Docs: format sys.float_info properly

### PR DESCRIPTION
**Pull request by erlend-aasland**
- Normalise capitalisation and punctuation
- Use .. attribute:: markup for named tuple attributes
- Use :c:macro: markup for C macros
- Use a list for the 'rounds' attribute values
- Use list-table, for better .rst readability

----
:books: Documentation preview :books:: https://cpython-previews--108107.org.readthedocs.build/

### CODE REVIEW

**Comment by erlend-aasland** [comment]
I really like the `list-table` markup.

File: Doc/library/sys.rst
Comment (AA-Turner) [inline Doc/library/sys.rst]:
```suggestion
      * - .. attribute:: float_info.epsilon
```

File: Doc/library/sys.rst
Comment (AA-Turner) [inline Doc/library/sys.rst]:
```suggestion
      * - .. attribute:: float_info.dig
```

File: Doc/library/sys.rst
Comment (AA-Turner) [inline Doc/library/sys.rst]:
```suggestion
      * - .. attribute:: float_info.mant_dig
```

File: Doc/library/sys.rst
Comment (AA-Turner) [inline Doc/library/sys.rst]:
```suggestion
      * - .. attribute:: float_info.max
```

File: Doc/library/sys.rst
Comment (AA-Turner) [inline Doc/library/sys.rst]:
```suggestion
      * - .. attribute:: float_info.max_exp
```

File: Doc/library/sys.rst
Comment (AA-Turner) [inline Doc/library/sys.rst]:
```suggestion
      * - .. attribute:: float_info.max_10_exp
```

File: Doc/library/sys.rst
Comment (AA-Turner) [inline Doc/library/sys.rst]:
```suggestion
      * - .. attribute:: float_info.min
```

File: Doc/library/sys.rst
Comment (AA-Turner) [inline Doc/library/sys.rst]:
```suggestion
      * - .. attribute:: float_info.min_exp
```

File: Doc/library/sys.rst
Comment (AA-Turner) [inline Doc/library/sys.rst]:
```suggestion
      * - .. attribute:: float_info.min_10_exp
```

File: Doc/library/sys.rst
Comment (AA-Turner) [inline Doc/library/sys.rst]:
```suggestion
      * - .. attribute:: float_info.radix
```

File: Doc/library/sys.rst
Comment (AA-Turner) [inline Doc/library/sys.rst]:
```suggestion
      * - .. attribute:: float_info.rounds
```

**Comment by rhettinger** [comment]
Perhaps also mention subnormals because this was a point of confusion in the past: https://github.com/python/cpython/issues/93488

**Comment by rhettinger** [comment]
Also see the comments and edits in https://github.com/python/cpython/issues/85988

**Comment by AA-Turner** [comment]
> Perhaps also mention subnormals

This PR is just about re-formatting the table such that it reads better, so I think adding any major new content would be out-of-scope? Will defer to Erlend, though.

A

**Comment by mdickinson** [comment]
+1 for the formatting changes; agreed that we can tweak the content if necessary in a separate PR.

**Review by mdickinson — APPROVED** [review]

**Comment by erlend-aasland** [comment]
> > Perhaps also mention subnormals
> 
> This PR is just about re-formatting the table such that it reads better, so I think adding any major new content would be out-of-scope? Will defer to Erlend, though.

Definitely; content changes are out of scope for this PR.

I'll add your Sphinx suggestions before landing.

Thanks for chiming in, y'all.

**Pull request merged**
